modules/bridge/src/main/ts/ephox/bridge/components/dialog/DialogFooterButton.ts

```diff
@@ -31,7 +31,15 @@ export interface DialogFooterMenuButtonSpec extends BaseDialogFooterButtonSpec {
   items: DialogFooterMenuButtonItemSpec[];
 }
 
-export type DialogFooterButtonSpec = DialogFooterNormalButtonSpec | DialogFooterMenuButtonSpec;
+export interface DialogFooterTogglableButtonSpec extends BaseDialogFooterButtonSpec {
```

Comment:
...TogglabelButton => ToggleButton (like in toolbar  addToggleButton)
DialogFooterTogglableButton => DialogFooterToogleButton
type: 'togglableButton'  => 'togglebutton' (with low case)
ect...

Reply:
done